lisa/operating_system.py

```diff
@@ -2177,10 +2179,75 @@ def name_pattern(cls) -> Pattern[str]:
 
 
 class SlMicro(Suse):
+    # all tests requiring packages/repos skipped due to transactional-update feature
     @classmethod
     def name_pattern(cls) -> Pattern[str]:
         return re.compile(r"^sl-micro$", re.IGNORECASE)
 
+    def add_azure_core_repo(
+        self, repo_name: Optional[AzureCoreRepo] = None, code_name: Optional[str] = None
+    ) -> None:
+        raise SkippedException(UnsupportedDistroException(self._node.os))
```

Comment:
For all skipped exception, add one more message to explain the reason, why SlMicro test cases are skipped as unsupported os.